### DISCUSSION TITLE
3D rendering on Apple Silicon

### DISCUSSION THREAD

**Discussion by sedghi** [nroduit/Weasis#507] [Q&A]
Congratulations on the release of Apple Silicon! It's looking really smooth. 
I've been trying to render a 512x512x135 CT in the 3D Viewer, but I keep encountering some errors/warnings. I believe my M2 Ultra can handle this rendering.
 
Do you have any recommendations on how to fix this?

![CleanShot 2024-01-09 at 09 50 37](https://github.com/nroduit/Weasis/assets/7490180/14c0d789-6d11-4e1c-8dcf-dbca32d57f1b)

**Comment by nroduit** [comment]
See [requirement](https://weasis.org/en/tutorials/dicom-3d-viewer/) in the documentation.
The OpenGL drivers 4.3 (released in 2012) is not available on Mac.

**Reply by sedghi** [reply]
Thanks for your answer, is there any plan to move away from OpenGL?

**Reply by nroduit** [reply]
No, because the frameworks for the successor Vulkan are not mature enough or not available for the current framework JOGL.
